rdflib/plugins/parsers/hext.py

```diff
@@ -24,14 +24,23 @@ def __init__(self):
         pass
 
     def _load_json_line(self, line: str):
-        return [x if x != "" else None for x in json.loads(line)]
+        # this complex handing is because the 'value' component is
+        # allowed to be "" but not None
+        # all other "" values are treated as None
+        ret1 = json.loads(line)
+        ret2 = [x if x != "" else None for x in json.loads(line)]
```

Comment:
```suggestion
        ret2 = [x if x != "" else None for x in ret1]
```

I'm just wondering, won't thins work equally well? I think it should, will test if I have a moment later tonight.

Reply:
Yes, oops, thanks